### ISSUE TITLE
iwanttfc.com: detection

### ISSUE THREAD

**Issue by JohnDoe01234** [uBlockOrigin/uAssets#19667]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.iwanttfc.com/`

### Category

detection

### Description

After selecting what show and episode to play, the adblock detection pops up.

### Other extensions used

None.

### Screenshot(s)

https://github.com/uBlockOrigin/uAssets/assets/144472828/d729d395-1b34-4874-9c72-f9adfb14bdda

### Configuration

<details>

```yaml
uBlock Origin: 1.51.0
Firefox Mobile: 117
filterset (summary):
 network: 100627
 cosmetic: 32094
 scriptlet: 21973
 html: 1227
listset (total-discarded, last-updated):
 added:
  ublock-annoyances: 5474-10, 2m
 default:
  user-filters: 0-0, never
  ublock-filters: 35740-350, 3m
  ublock-badware: 7951-1, 3m
  ublock-privacy: 577-1, 3m
  ublock-unbreak: 2182-1, 3m
  ublock-quick-fixes: 263-0, 3m
  adguard-mobile: 8624-124, 2m
  easylist: 70374-16678, 2m
  easyprivacy: 33208-725, 2m
  urlhaus-1: 5849-0, 2m
  plowe-0: 3723-1, 2m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 9575 ms
 maxAssetCacheWait: 215 ms
popupPanel:
 blocked: 12
 network:
  amplitude.com: 1
  crwdcntrl.net: 1
  datazoom.io: 1
  facebook.com: 1
  googlesyndication.com: 2
  googletagmanager.com: 1
  imasdk.googleapis.com: 2
  moatads.com: 2
  moengage.com: 1
```
</details>

**Comment by ghajini** [comment]
does it require login/subscription?

**Comment by mapx-** [comment]
test

`iwanttfc.com##+js(set, settings.adBlockDetectionEnabled, false)`

**Comment by mapx-** [comment]
@ghajini 

try in the `Recently Added` row, this one:
![image](https://github.com/uBlockOrigin/uAssets/assets/10303732/20b172d2-4c78-4a71-ada3-f159dce82e72)

**Comment by ghajini** [comment]
> test
> 
> `iwanttfc.com##+js(set, settings.adBlockDetectionEnabled, false)`
> 

that seems to work👍

**Comment by JohnDoe01234** [comment]
Thank you all :)

**Comment by iamflorencejay** [comment]
It is not working, how to fix this now?

**Comment by GrumpyGorgg** [comment]
<img width="1362" height="517" alt="Image" src="https://github.com/user-attachments/assets/6a407ef6-72aa-4d5d-ab00-1b05a5caa368" />

Good day! I'm having a similar problem, tried using this one "iwanttfc.com##+js(set, settings.adBlockDetectionEnabled, false)" on my filter it still doesn't work.

**Comment by mapx-** [comment]
> similar problem

same page ? provide the exact link where you get the issue

**Comment by GrumpyGorgg** [comment]
> > similar problem
> 
> same page ? provide the exact link where you get the issue

Hello! After selecting what show and episode to play, the adblock detection pops up. Below is the exact link that is showing the issue. Thank you and have a good day!

`https://www.iwanttfc.com/player/cc61a3a0-aa13-4b2d-83b8-219c238f1d47`

**Comment by uBlock-user** [comment]
@GrumpyGorgg fixed in https://github.com/uBlockOrigin/uAssets/commit/7c8166b48a2dd63dff7311e3718248a2f05e43b9

**Comment by GrumpyGorgg** [comment]
> [@GrumpyGorgg](https://github.com/GrumpyGorgg) fixed in [7c8166b](https://github.com/uBlockOrigin/uAssets/commit/7c8166b48a2dd63dff7311e3718248a2f05e43b9)

It's working pretty well, thank you so much!
